substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/configure/SerializationConfigurationParser.java

```diff
@@ -58,9 +59,10 @@ public void parseAndRegister(Reader reader) throws IOException {
     }
 
     private void parseSerializationDescriptorObject(Map<String, Object> data) {
-        checkAttributes(data, "serialization descriptor object", Collections.singleton(NAME_KEY), Arrays.asList(CUSTOM_TARGET_CONSTRUCTOR_CLASS_KEY, CONDITIONAL_KEY));
+        checkMultipleOptionsAttributes(data, "serialization descriptor object", Collections.unmodifiableCollection(Arrays.asList(NAME_KEY, LAMBDA_CAPTURING_CLASS_KEY)),
```

Comment:
`Multiple` is not accurate. They are exclusive options so maybe `checkExclusiveOptionsAttributes`.

This is also not overly general as it supports only a single option.

Reply:
> `Multiple` is not accurate. They are exclusive options so maybe `checkExclusiveOptionsAttributes`.
> 
> This is also not overly general as it supports only a single option.

Changed as suggested.

Reply:
> What happened to `Arrays.asList(CUSTOM_TARGET_CONSTRUCTOR_CLASS_KEY, CONDITIONAL_KEY)`?

It's been removed by mistake.